chore: re-generated to pick up changes from self

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/7cd5954a-9bd8-4be1-8def-21085a41d0cc/targets

- [ ] To automatically regenerate this PR, check this box.

dinagraves commented:
Regenerating.  It should not replace flaky bot with build cop.

Shabirmean commented:
The `Kokoro docs-presubmit` failure seems to be originating from the sphinx library. The repo for the library has a comment (https://github.com/sphinx-doc/sphinx/issues/5870#issuecomment-468285753) saying this was fixed in v1.7.6 of the library. 

It seems  like the sphinx version used by this repo is an older version 1.5.5 (https://github.com/googleapis/python-container/blob/master/noxfile.py#L204)

@busunkim96 has filed an issue on behalf of this in the [docs-templates repo](https://github.com/googleapis/doc-templates/issues/94). It's not top priority but is in the pipeline to be fixed eventually.

Shabirmean commented:
The `Kokoro docs-presubmit` failure seems to be originating from the sphinx library. The repo for the library has a comment (https://github.com/sphinx-doc/sphinx/issues/5870#issuecomment-468285753) saying this was fixed in v1.7.6 of the library. 

It seems  like the sphinx version used by this repo is an older version 1.5.5 (https://github.com/googleapis/python-container/blob/master/noxfile.py#L204)

@busunkim96 has filed an issue on behalf of this in the [docs-templates repo](https://github.com/googleapis/doc-templates/issues/94). It's not top priority but is in the pipeline to be fixed eventually. Until then the `docs-presubmit` checks on Kokoro would fail

yoshi-automation committed:
changes without context

        autosynth cannot find the source of changes triggered by earlier changes in this
        repository, or by version upgrades to tools such as linters.
